drivers/chroot_windows.go

```diff
@@ -0,0 +1,16 @@
+package graphdriver
```

Comment:
Shouldn't this be +build windows?

Reply:
[For a while now](https://golang.org/doc/go1.4#filenames), that's been implicit when the source file's base name ends with `_` followed by an architecture or operating system name.

Reply:
Right _windows.go indicates only build this on Windows
_linux.go only on Linux.